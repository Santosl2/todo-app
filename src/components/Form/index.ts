export * from "./Input";
export * from "./MainForm";
export * from "./Button";
