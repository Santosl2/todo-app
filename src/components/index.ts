export * from "./Header";
export * from "./Form";
export * from "./Badge";
export * from "./TextWithBadge";
export * from "./Item";
export * from "./NoResults";
