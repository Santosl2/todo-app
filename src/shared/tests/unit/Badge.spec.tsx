import { Badge } from "@/components";
import { render, screen } from "@testing-library/react";

describe("Badge", () => {
  it("should render correctly", () => {
    render(<Badge>Test 1</Badge>);

    expect(screen.getByText("Test 1")).toBeInTheDocument();
  });

  it("match snapshot", () => {
    render(<Badge>Test 1</Badge>);

    expect(screen.getByText("Test 1")).toMatchSnapshot();
  });
});
