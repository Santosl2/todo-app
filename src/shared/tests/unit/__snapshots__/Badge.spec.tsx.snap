// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge match snapshot 1`] = `
<span
  class="rounded-full bg-gray-400 px-2 text-gray-200"
>
  Test 1
</span>
`;
